IndexHelper.js

```diff
@@ -223,4 +239,19 @@ export class IndexHelper {
     const data = new TextEncoder().encode(value);
     return this.hmac.sign({data});
   }
+
+  _formatAttribute(attribute) {
+    const keys = split(attribute);
+    const scope = keys[0];
+    keys.shift();
+    const key = keys.join('.');
+    if(!VALID_SCOPES.includes(scope)) {
+      throw new Error('Expected attribute prefixed with "content" or ' +
```

Comment:
Yeah, this sort of text is better for the error message above, crib from it appropriately.